packages/ci/src/utils/test-run.ts

```diff
@@ -120,33 +120,39 @@ function draw(testRuns: Heroku.TestRun[], watchOption = false, count = 15) {
     )
   })
 
-  cli.table(data, {
-    printHeader: undefined,
-    columns: [
-      {key: 'iconStatus', width: 1, label: ''}, // label '' is to make sure that widh is 1 character
-      {key: 'number', label: ''},
-      {key: 'branch'},
-      {key: 'sha'},
-      {key: 'status'}
-    ]
-  })
+  if (jsonOption) {
+    cli.styledJSON(testRuns)
+  } else {
+    cli.table(data, {
```

Comment:
@talbright I recommend moving:
```
   let data: any = []
 
   latestTestRuns.forEach(testRun => {
     data.push(
       {
         iconStatus: `${statusIcon(testRun)}`,
         number: testRun.number,
         branch: testRun.commit_branch,
         sha: testRun.commit_sha!.slice(0, 7),
         status: testRun.status
       }
    )
  })
```
Inside the the`else` since `data` should only be scoped to this branch.

Reply:
Taking a look at that exposes an issue with the code here: https://github.com/heroku/cli/blob/ef110ad2e257cc59102cd2af54d9c32e8c77b709/packages/ci/src/utils/test-run.ts#L138-L140 

If we are printing json that wouldn't be the correct amount to scroll-up. Now that I think about it, what if we ignore the watch option for json, and just short-circuit with a return at the top of the function if the format is json?

```typescript
if (jsonOption) {
  cli.styleJSON(testRuns)
  return
}
```

For most cases JSON would be used as input into another command (as is my case), and is going to be to long to be watchable.